utils/ManyLinux1_64bit/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pyca/cryptography-manylinux1:x86_64
+FROM quay.io/pypa/manylinux1_x86_64:latest
```

Comment:
we could drop this dockerfile since we decided not to support manylinux1 for python 3.10